phpstan.neon

```diff
@@ -17,3 +17,6 @@ parameters:
 		- '#Call to an undefined method DOMNode::getTagName\(\)\.#'
 		- '#Return type \(void\) of method Symfony\\Component\\Panther\\DomCrawler\\Crawler::clear\(\) should be compatible with return type \(Facebook\\WebDriver\\WebDriverElement\) of method Facebook\\WebDriver\\WebDriverElement::clear\(\)#'
 		- '#Method Symfony\\Component\\Panther\\DomCrawler\\Crawler::getIterator\(\) should return ArrayIterator&iterable<DOMNode> but returns ArrayIterator<\(int\|string\), Facebook\\WebDriver\\WebDriverElement>\.#'
+		- '#PHPDoc type array of property Symfony\\Component\\Panther\\DomCrawler\\Field\\FileFormField::\$value is not covariant with PHPDoc type string of overridden property Symfony\\Component\\DomCrawler\\Field\\FormField::\$value#'
```

Comment:
I don't really get this one, in parent class method `getValue` is type-hinted as `array|string`, but field `$value` is type-hinted as `string` exclusively. We could make `$value` hold a json-encoded array or something, but I feel like it's some inconsistency to be fixed in parent symfony class.